omnibridge/cli/argument_parser/commands.py

```diff
@@ -1,45 +1,47 @@
-from enum import Enum
-from typing import Dict, Any, Callable
+from typing import Dict, Any
+from omnibridge.flows.branching_flow import BranchingFlow
+from omnibridge.flows.sequential_flow import SequentialFlow
 from omnibridge.wrappers.api_key import ApiKey
 from omnibridge.saved_data.json_data_manager import JsonDataManager
 from omnibridge.wrappers.wrapper_instances.gpt_wrapper import GPTWrapper
 from omnibridge.wrappers.wrapper_instances.dalle_wrapper import DALLEWrapper
-
-from omnibridge.wrappers.wrapper_instances.type_name_to_wrapper import type_names
-
-class Commands(str, Enum):
-    ADD_KEY = "add-key"
-    ADD_CHATGPT = "add-chatgpt"
-    ADD_DALLE = "add-dalle"
-    LIST_WRAPPERS = 'list-wrappers'
+from omnibridge.wrappers.wrapper_instances.type_name_to_wrapper import ModelLoader
 
 
 def add_key(args: Dict[str, Any], file_path: str = '') -> None:
     api_key = ApiKey(args['value'])
     JsonDataManager.save(["api keys", args['name']], api_key, file_path=file_path)
 
 
-def add_chatgpt(name: str, args: Dict[str, Any], file_path: str = '') -> None:
+def add_chatgpt(args: Dict[str, Any], file_path: str = '') -> None:
+    print(args)
+    name = args['name']
     api_key: ApiKey = JsonDataManager.load(["api keys", args['key']], ApiKey, file_path=file_path)
-    wrapper: GPTWrapper = GPTWrapper(name, api_key.value, args['model'])
-    JsonDataManager.save(["models", args['name']], wrapper, file_path=file_path)
+    wrapper: GPTWrapper = GPTWrapper(name, api_key.value, args['sub_model'])
```

Comment:
comparing "add_chatgpt" and "add_dalle", all the lines are code duplication other than this one, creating the wrapper. getting the duplicated code into the caller function will be better, with the api_key line being bit tricky, although a code duplication, not all wrappers will use it. need to think about that.